product_code_builder/models/product_attribute_value.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
+# © 2015 Benoît GUILLOT <benoit.guillot@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProductAttributeValue(models.Model):
+    _inherit = "product.attribute.value"
+    _code = "code"
```

Comment:
What does this line do? I'm trying to port to 10 but can find no usage of it anywhere in odoo